README.md

```diff
@@ -63,11 +63,11 @@ Developers choose Citus for two reasons:
 
 ## Getting Started
 
-The quickest way to get started with Citus is to use the [Hyperscale (Citus)](https://docs.microsoft.com/azure/postgresql/quickstart-create-hyperscale-portal) deployment option in the Azure Database for PostgreSQL managed service—or [set up Citus locally](https://docs.citusdata.com/en/stable/installation/single_node.html).
+The quickest way to get started with Citus is to use the [Azure Cosmos DB for PostgreSQL](https://learn.microsoft.com/en-us/azure/cosmos-db/postgresql/quickstart-create-portal) deployment option in the Azure Cosmos DB for PostgreSQL managed service—or [set up Citus locally](https://docs.citusdata.com/en/stable/installation/single_node.html).
```

Comment:
A few things to adjust:
1) Let's remove the /en-us/ locale from the docs aka learn URLs, since we don't want to hardcode URLs to the English locale
2) it's no longer a deployment option! 
3) oops, seems there's a repetition above, it now says "Azure Cosmos DB for PostgreSQL deployment option in the Azure Cosmos DB for PostgreSQL managed service" // How about, instead: 

"[Azure Cosmos DB for PostgreSQL](XXX link) managed service in the cloud—or..."

Reply:
> Let's remove the /en-us/ locale from the docs aka learn URLs, since we don't want to hardcode URLs to the English locale

Ah, makes sense

> it's no longer a deployment option!

right, missed that :/ 

> oops, seems there's a repetition above,

Fixed that